com.unity.netcode.gameobjects/Editor/NetworkObjectEditor.cs

```diff
@@ -114,10 +114,34 @@ public override void OnInspectorGUI()
             {
                 EditorGUI.BeginChangeCheck();
                 serializedObject.UpdateIfRequiredOrScript();
+                var ownershipProperty = serializedObject.FindProperty(nameof(NetworkObject.Ownership));
```

Comment:
Is it worth putting a comment above this section explaining that it's for the `OwnershipStatus` flags?